main.go

```diff
@@ -84,7 +86,7 @@ func registerExporter(namespace string, nginxUrls []string, nginxPlusUrls []stri
 	prometheus.MustRegister(exporter.NewNginxPlusExporter(
 		client,
 		scraper.NewNginxScraper(),
-		scraper.NewNginxPlusScraper(),
+		scraper.NewNginxPlusScraper(excludeUpstreamAddresses),
```

Comment:
Let’s find another approach for initialization of this structure. 
Check jaeger tracing for good examples